controllers/coursegroups.js

```diff
@@ -155,6 +155,8 @@ router.get('/:courseGroupId/', (req, res, next) => {
 		_.each(courseGroup.userIds, u => u.displayName = `${u.firstName} ${u.lastName}`);
 		_.each(course.teacherIds, t => t.displayName = `${t.firstName} ${t.lastName}`);
 
+		const showRoomView = Configuration.get('ROOM_VIEW_ENABLED') || false;
```

Comment:
this should be added to the config json, otherwise is always false
https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/config/default.json

also, should test if overwrite the config from env var with something like ROOM_VIEW_ENABLED=false
in some case, if the type is string and not boolean, it might evaluate "false" as being true

Reply:
configuration should be extensively tested in other places, especially the handling of the string "false". That is responsability of the configuration service itself.
This flag in particular already exists in the client, so it doesnt need to be added again.

apart from that the || false is redundant, since Configuration.get('ROOM_VIEW_ENABLED') has a default value (false), and thus will never be undefined.

Reply:
ah, my bad. I checked in wrong file. You're right about the part of the testing, just was thinking to avoid potential general problem